src/components/Sidebar/Sidebar.minors.tsx

```diff
@@ -24,17 +26,24 @@ export function Item({
         onClick={onClick}
         size="md"
         variant="minimalTransparent"
+        isActive={isActive}
+        backgroundColor={activeColor}
         {...props}
       />
     </Tip>
   );
 }
 
-const ItemStyled = styled(Button)`
+const ItemStyled = styled(Button)<{
+  isActive?: boolean;
+  backgroundColor?: string;
+}>`
   height: 3rem;
   text-align: left;
   padding: 0.25rem 0.25rem 0.25rem 0.75rem;
   justify-content: start;
+  ${({ isActive, backgroundColor }) =>
+    isActive ? `background-color: ${backgroundColor}` : ''};
```

Comment:
What if the designs also call for more than just the background color change?

Reply:
Our designers use the iris design as a pattern. 

I only added active state color, because we dont have it in current version at all

Reply:
Sorry, I probably wasn't being clear with my question: my concern with adding _just_ a background color change is that it is very opinionated. What if a designer requests for a font color change as well? Or they eventually want to scale the entire container? I think a more flexible approach would be to pass in a style object that relies on your `isActive` state.

Reply:
Fixed, can you please check